Sources/Fullscreen/FavoriteAdsList/FavoriteAdsListView.swift

```diff
@@ -306,6 +310,22 @@ extension FavoriteAdsListView: UITableViewDelegate {
 
         return configuration
     }
+
+    public func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        if sendScrollUpdates {
```

Comment:
Can we use `!isEditing` instead?

Reply:
I used that initially, but this value needs to be set at different times within the `setEditing(_:)` method for this to work correctly. I decided to change the name of the variable to avoid confusion.